cmd/server/accounts/router.go

```diff
@@ -148,6 +150,39 @@ func createCustomerAccount(logger log.Logger, repo Repository, fedClient fed.Cli
 	}
 }
 
+func getCustomerAccountByID(logger log.Logger, repo Repository, fedClient fed.Client) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		w = route.Responder(logger, w, r)
+		customerID := route.GetCustomerID(w, r)
+		accountID := getAccountID(w, r)
+
+		account, err := repo.getCustomerAccount(customerID, accountID)
+		if err != nil {
+			moovhttp.Problem(w, err)
+			return
+		}
+
+		if account == nil {
+			moovhttp.Problem(w, fmt.Errorf("account with customerID=%s and accountID=%s not found", customerID, accountID))
+			return
+		}
+
+		details, err := fedClient.LookupInstitution(account.RoutingNumber)
+		if err != nil {
+			moovhttp.Problem(w, fmt.Errorf("looking up institution details: %v", err))
+			return
+		}
+
+		if details != nil {
+			account.Institution = *details
+		}
+
+		w.Header().Set("Content-Type", "application/json; charset=utf-8")
+		w.WriteHeader(http.StatusOK)
+		json.NewEncoder(w).Encode(account)
```

Comment:
In other parts of the code, we ignore the error from `Encode` but shouldn't we handle it?

Reply:
I wouldn't. When I was looking at that earlier there wasn't a reasonable time we'd error without noticing the response JSON was invalid. Do you know any reason to check this error?

Reply:
I can't think of any reasons at the moment, but I'd usually handle it to avoid any elusive bugs.

Reply:
I faced such error once... it was a pain to find it :D